path_login.go

```diff
@@ -55,19 +61,16 @@ func (b *jwtAuthBackend) pathLogin(ctx context.Context, req *logical.Request, d
 		return nil, err
 	}
 	if role == nil {
-		return logical.ErrorResponse("role could not be found"), nil
+		return logical.ErrorResponse("role '%s' could not be found", roleName), nil
```

Comment:
I just learned from Jeff that we can replace ```'%s'``` with `%q` and it will come out the same!

Reply:
Oh, I see you use `%q` below so there's probably some nuance of this that I don't yet know.